### PR TITLE
[Bugfix] YAxis Labels Width | Chart | Dashboard

### PR DESCRIPTION
**Pull request by Civolilah**
@beganovich @turbo124 The PR includes the resolution of a bug for the YAxis label when dealing with long numbers. I have now implemented a solution that ensures a consistently dynamic width. Screenshot:

Screenshot 2023-08-07 at 20 18 07

Let me know your thoughts.

### CODE REVIEW

File: src/pages/dashboard/components/Chart.tsx [hunk 1]
```diff
@@ -90,6 +90,24 @@ export function Chart(props: Props) {
...
+          currency
+        ).toString().length;
+
+        return Math.max(currentMax, currentTickLength);
+      }, maxTick);
+    }, 0);
+
+    return largestTick;
+  }, [chartData]);
+
```
Comment (Civolilah) [inline src/pages/dashboard/components/Chart.tsx:110]: @beganovich After conducting research and reviewing the opinions of other individuals on the GitHub issue regarding the same problem, I am quite confident that the package does not provide a solution for this. Therefore, we need to develop a custom solution. Let me know your thoughts.

Comment (beganovich) [reply src/pages/dashboard/components/Chart.tsx:110]: That's fine. @turbo124 can you QA this?

**Comment by turbo124** [comment]
@beganovich 

LGTM

**Review by beganovich — APPROVED** [review]
Looks good, thanks 👍️

**Pull request merged**